src/main/java/nl/dtls/fairdatapoint/api/dto/shape/ShapeChangeDTO.java

```diff
@@ -38,6 +39,9 @@
     @NotBlank
     private String name;
 
+    @NotNull
+    private Boolean published;
```

Comment:
Object vs primitive? Or is `null` used as a third state?

Reply:
Fixed, not it is a primitive.